src/engraving/dom/segmentlist.cpp

```diff
@@ -48,11 +48,10 @@ SegmentList SegmentList::clone() const
 
 Segment* SegmentList::firstActive() const
 {
-    Segment* segment = m_first;
-    if (segment->isActive()) {
-        return segment;
+    if (Segment* segment = m_first) {
```

Comment:
Is it actually ok for m_first to be null? Cause that means that the Measure contains no Segments at all. Do we actually allow that? Is it just a temporary state?